types/src/lib.rs

```diff
@@ -23,38 +23,6 @@ extern crate alloc;
 #[macro_use]
 extern crate std;
 
-mod access_rights;
```

Comment:
I thought the order was mods then imports?

Reply:
Whoops, CLion did this.  Will fix.